src/main/java/codeu/controller/AdminServlet.java

```diff
@@ -22,13 +22,17 @@
 import java.io.IOException;
 import java.time.Instant;
 import java.util.List;
+import java.util.Properties;
 import java.util.UUID;
+
+import javax.mail.Authenticator;
+import javax.mail.PasswordAuthentication;
+import javax.mail.Session;
```

Comment:
These new imports look unused, perhaps this file is also not supposed to be in this PR?